integration-tests/client/chainlink_models.go

```diff
@@ -772,6 +772,70 @@ minIncomingConfirmations = {{.MinIncomingConfirmations}}
 	return marshallTemplate(k, "Keeper Job", keeperTemplateString)
 }
 
+// AutoOCR2JobSpec represents a V2 keeper spec
+type AutoOCR2JobSpec struct {
+	ContractID         string `toml:"contractID"`
+	OCRKeyBundleID     string `toml:"ocrKeyBundleID"`
+	TransmitterID      string `toml:"transmitterID"`
+	P2Pv2Bootstrappers string `toml:"p2pv2Bootstrappers"`
+	ChainID            int    `toml:"chainID"`
+}
+
+// Type returns the type of the job
+func (k *AutoOCR2JobSpec) Type() string { return "keeper" }
+
+// String representation of the job
+func (k *AutoOCR2JobSpec) String() (string, error) {
```

Comment:
Does `func (o *OCR2TaskJobSpec) String() (string, error) {` not cover this job template already?

Reply:
ill try it with those